packages/twenty-front/src/modules/ui/navigation/bread-crumb/components/EditableBreadcrumbItem.tsx

```diff
@@ -0,0 +1,120 @@
+import { TextInputV2 } from '@/ui/input/components/TextInputV2';
+import { EditableBreadcrumbItemHotkeyScope } from '@/ui/navigation/bread-crumb/types/EditableBreadcrumbItemHotkeyScope';
+import { useScopedHotkeys } from '@/ui/utilities/hotkey/hooks/useScopedHotkeys';
+import { useListenClickOutside } from '@/ui/utilities/pointer-event/hooks/useListenClickOutside';
+import styled from '@emotion/styled';
+import { useRef, useState } from 'react';
+import { Key } from 'ts-key-enum';
+import { isDefined } from 'twenty-ui';
+import { useHotkeyScopeOnMount } from '~/hooks/useHotkeyScopeOnMount';
+
+type EditableBreadcrumbItemProps = {
+  isRenaming: boolean;
+  className?: string;
+  setIsRenaming: (isRenaming: boolean) => void;
+  defaultValue: string;
+  placeholder: string;
+  onSubmit: (value: string) => void;
+  onCancel: (value: string) => void;
+  onClickOutside: (event: MouseEvent | TouchEvent, value: string) => void;
+  hotkeyScope: string;
+};
+
+const StyledInput = styled(TextInputV2)`
+  background-color: white;
```

Comment:
what is it from a design perspective:
1. should we add a props to the component?
2. what is white? should be part of the theme